test/e2e/parallel/registration_service_test.go

```diff
@@ -781,17 +781,16 @@ func TestUsernames(t *testing.T) {
 	t.Run("get usernames 200 response", func(t *testing.T) {
 		// given
 		// we have a user in the system
-		_, mur := NewSignupRequest(awaitilities).
+		user := NewSignupRequest(awaitilities).
 			Username("testgetusernames").
 			Email("testgetusernames@redhat.com").
 			ManuallyApprove().
 			TargetCluster(awaitilities.Member1()).
 			EnsureMUR().
 			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
 			NoSpace().
-			Execute(t).
-			Resources()
-
+			Execute(t)
+		mur := user.MUR
```

Comment:
the same, no need to create another variable since the mur is used only once 4 lines below